docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Use an official PHP-FPM base image
+FROM php:8.4-fpm
+
+# Install required packages and PHP extensions
+RUN apt-get update && apt-get install -y \
+    nginx \
+    curl \
+    zip \
+    unzip \
+    git \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    libzip-dev \
+    supervisor \
+    wget \
+    && docker-php-ext-install \
+        mbstring \
+        exif \
+        pcntl \
+        bcmath \
+        gd \
+        zip \
+        opcache \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install confd
+RUN wget https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.16.0-linux-amd64 \
+    && mv confd-0.16.0-linux-amd64 /usr/local/bin/confd \
```

Comment:
do you think this could use `$(uname -m)` everywhere that `amd64` exists?

Reply:
The only thing I can do here is to build it from the source, its annoying to have this extra step, but not all architectures are available.